stylesheets/Senior-Basic.scss

```diff
@@ -393,10 +399,120 @@ button.close:active:hover {
   margin: -20px 0;
 }
 
-/* Reduce size for XS screens */
-@media (max-width: 375px) {
-  #thank-you-text,
-  #workflow-background-wrapper {
-    width: 85% !important;
+/* Add border to photo guide */
+@media (max-width: 799px) {
+  .picture-guide {
+    border: 1px solid #e9eaed !important;
```

Comment:
suggestion (non-blocking): Since this border rule is identical across all sizes, if you wanted you could move it out and define it once for `.picture-guide` outside of any `@media` blocks.

Reply:
@arkadyan Good idea. Implemented and ready for review.